webpack/base.js

```diff
@@ -4,14 +4,17 @@ const nodePath = path.resolve(__dirname, '../node_modules');
 const commonPath = path.resolve(__dirname, '../common');
 const appPath = path.resolve(__dirname, '../src');
 
+const testPath = path.resolve(__dirname, '../test/config');
+
 const base = {
   // Allows for absolute paths from locations indicated in 'root'
   resolve: {
     modules: [
       nodePath,
       appPath,
-      commonPath
-    ]
+      commonPath,
+      testPath,
```

Comment:
I assume this is just for Karma. From my recollection Jest has it's own module paths.

https://github.com/CodeSpaceHQ/telescreen/blob/master/sub-projects/hub/jest.server.js

Reply:
Correct. and again, webpack will change all together